pkg/deploy/controller/deploymentconfig/factory.go

```diff
@@ -40,7 +40,7 @@ func (factory *DeploymentConfigControllerFactory) Create() controller.RunnableCo
 		},
 	}
 	queue := cache.NewFIFO(cache.MetaNamespaceKeyFunc)
-	cache.NewReflector(deploymentConfigLW, &deployapi.DeploymentConfig{}, queue, 2*time.Minute).Run()
+	cache.NewReflector(deploymentConfigLW, &deployapi.DeploymentConfig{}, queue, 30*time.Second).Run()
```

Comment:
Ok, got this working on top of https://github.com/openshift/origin/pull/9002 without the need to reduce the interval. Now all I need is the rebase to land so we can start using shared caches @mfojtik :)